lib/std/fs.zig

```diff
@@ -899,10 +900,11 @@ pub const IterableDir = struct {
                                 .dirname_len = self.name_buffer.items.len,
                             });
                             top = &self.stack.items[self.stack.items.len - 1];
```

Comment:
An alternate fix would be to just do `- 2` here, since `top` the only usage of `top` after this assignment is in the return statement below, but I think having a separate `containing` makes the intentions clearer.